src/app/view/perspectives/calendar/events/header/calendar-header.component.html

```diff
@@ -0,0 +1,57 @@
+<div class="row text-center">
+  <div class="col-lg-4 d-lg-flex align-items-lg-center justify-content-lg-start">
+    <div class="btn-group">
+      <div class="btn btn-primary"
+           mwlCalendarPreviousView
+           [view]="mode"
+           [viewDate]="viewDate"
+           (viewDateChange)="onViewDateChange($event)"
+           i18n="@@previous">
```

Comment:
Maybe it would make sense to provide a more specific translation ID here. We might want to have multiple variants of text `Previous` in the application which might be translated differently, use different case, etc.